src/App.tsx

```diff
@@ -38,9 +39,7 @@ const PageContent = () => {
       <Route exact path={AppPaths.SIGN_IN}>
         <SignIn />
       </Route>
-      <Route exact path={AppPaths.CALC}>
-        <Calculator/>
-      </Route>
+      <PrivateRoute exact path={AppPaths.CALC} component={Calculator} />
```

Comment:
Make these consistent if possible (I didn't check the doc of `<PrivateRoute>`)

Either all of them use `component` or make the pages to be the children.

The pros of using `component` is that t is short, but the cons will be that you can't add any additional elements because `component` only takes a single JSX element. Using children makes the code a little bit longer, but you will be able to fill in additional components if needed.

I personally will use the children way for flexibility in the future.